styles.css

```diff
@@ -300,6 +301,132 @@ main {
   color: rgba(91, 89, 87, 1);
 }
 
+.modal {
+  width: 90%;
+  height: 75vh;
+  padding: 4%;
+  background: #fff;
+  transform: scale(0);
+  transition: 200ms ease-in-out;
+  z-index: 10;
+  position: fixed;
+  top: 0;
+  left: 5%;
+}
+
+.modal.active {
+  transform: scale(1);
+}
+
+.modal-header {
+  display: flex;
+  justify-content: flex-start;
+  gap: 40px;
+  width: auto;
+  height: auto;
+  padding-left: 0;
+  align-items: flex-start;
+  background-color: inherit;
+}
+
+.modal-heading {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 22px;
+  color: #172b4d;
+}
+
+.modal-heading.font-size {
+  font-size: 1.3em;
+}
+
+.close-btn {
+  width: 20px;
+  height: 20px;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  justify-content: space-between;
+}
+
+.rotator1 {
+  width: 100%;
+  height: 2px;
+  background-color: #67798e;
+  transform: rotate(-45deg) translate(-6px, 6px);
+}
+
+.rotator2 {
+  width: 100%;
+  height: 2px;
+  background-color: #67798e;
+  transform: rotate(45deg) translate(-6px, -7px);
+}
+
+.tech-box.popup {
+  color: #3c3a39;
+  margin-bottom: 5%;
+  margin-top: 15%;
+}
+
+#img10 {
+  width: 100%;
+  height: 240px;
+  background: url(Images/img10.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+p.description.popup {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 10px;
+}
+
+.button-container {
+  display: flex;
+  gap: 20px;
+}
+
+#seeLive {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+  width: 45%;
+}
+
+#seeSource {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+  width: 50%;
+}
```

Comment:
- [ ] Kindly make sure you stick to the template as much as possible. the color, spacing(margin, padding) should be similar to the template in Figma. As you may see the bg-color of your button and the one in figma wich are quite different.
- [ ] Also consider having the same spacing(padding, margin) all around the content of the pop up as is the case on the original design.

### Your design :point_down:

![Screenshot from 2023-03-09 16-11-32](https://user-images.githubusercontent.com/97100091/224058547-9df203f6-2162-4c4d-87be-2a4e3c6d2dbe.png)

### Figma template :point_down: 

![image](https://user-images.githubusercontent.com/97100091/224058965-b0f54436-19e7-4565-89a6-9b780a93c68a.png)